src/common/components/play/GameMode.tsx

```diff
@@ -6,52 +6,54 @@ import PaperButton from '../PaperButton';
 interface GameModeProps {
   name: string
   imagePath?: string
+  explanation?: string
   modesPerRow?: number
   compact?: boolean
   disabled?: boolean
+  wrapper?: (inner: JSX.Element) => JSX.Element
   onSelect?: () => void
 }
 
 export default class GameMode extends React.Component<GameModeProps> {
   public render(): JSX.Element {
-    const { modesPerRow, compact, disabled, onSelect } = this.props;
-    const widthPercent = `${100 / (modesPerRow || 2)}%`;
-    const widthPadding = (((modesPerRow || 2) - 1) * 20) / (modesPerRow || 2);
+    const { modesPerRow, compact, disabled, wrapper, onSelect } = this.props;
+    const widthPercent = `${100 / (modesPerRow || 1)}%`;
+    const widthPadding = (((modesPerRow || 1) - 1) * 20) / (modesPerRow || 1);
 
     return (
       <PaperButton
         disabled={disabled}
         onClick={onSelect || noop}
         style={{
           width: `calc(${widthPercent} - ${widthPadding}px)`,
-          height: compact ? 'auto' : 230,
+          height: compact ? 'auto' : 100,
           position: 'relative',
-          marginBottom: 20
+          marginBottom: 20,
+          padding: compact ? 0 : '20px 0'
         }}
       >
-        {this.renderInner()}
+        {wrapper ? wrapper(this.renderInner()) : this.renderInner()}
       </PaperButton>
     );
   }
 
   private renderInner(): JSX.Element {
-    const { name, compact, disabled, imagePath } = this.props;
+    const { name, imagePath, explanation, compact, disabled } = this.props;
     return (
       <div
         style={{
-          filter: disabled ? 'blur(2px)' : undefined,
           padding: compact ? 15 : '0 20px',
           display: 'flex',
-          flexDirection: 'column',
+          flexDirection: 'row',
           justifyContent: 'space-between',
-          alignItems: 'center'
+          alignItems: 'center',
         }}
       >
         {imagePath && (
           <div
             style={{
-              height: 165,
-              width: '100%',
```

Comment:
This seems like a big change? 100% -> 100px?

Reply:
width is going from 100% to 200px because we're showing the image and text on the same row now